editor/events/overview.mdx

```diff
@@ -4,6 +4,10 @@ sidebarTitle: 'Overview'
 description: ''
 ---
 
+<Warning>
```

Comment:
Should this be an info (or something else) instead of a warning?

Reply:
Yeah, that seems fine. Especially if we're going to add a "Legacy" section to the sidebar. Done.